lua/Connector.lua

```diff
@@ -1,6 +1,16 @@
 -- original file found in a GPLv3 code repository, unclear if this is the intended license nor who the authors are
 -- SNI modifications by Berserker, jsd1982; modifications licensed under MIT License
 -- version 3 changes Read response from JSON to HEX
+-- lua 5.1/5.4 shim by zig; modifications licensed under MIT and WTFPL
+
+function get_lua_version()
+    local major, minor = _VERSION:match("Lua (%d+)%.(%d+)")
+    assert(tonumber(major) == 5)
+    if tonumber(minor) >= 4 then

```

Comment:
Lua's "minor" version numbers are generally full of breaking changes; they don't seem to subscribe to semver. Given that, I would just stick with a `==` comparison here since we have no idea what future changes would mean.